### PR TITLE
activate-ux: Master Key Confirm form UX enhancements

### LINKED ISSUE #673: L2 Invite flow: master ticket needs some polish
- [x] When copying the master ticket, there shouldn't be a sig (`~`) at the front. The form where you input it doesn't accept a sig. 
- [x] Pasting the master ticket into the form should paste each of the four segments into the appropriate boxes
- [x] Entering the correct number of characters into a master ticket input should advance the cursor to the next input

### LINKED ISSUE #700: Activation Screen : Confirm Backup needs an Error State
Step 2 Filled Error

### PR DESCRIPTION
**Pull request by tomholford**
# Preview

## Pasting with a ticket in the clipboard
*it pastes the ticket segments into the corresponding fields*

https://user-images.githubusercontent.com/16504501/140629934-68406a67-b651-46c0-9ccf-a079e61d6634.mov

## Pasting without a ticket in the clipboard
*it pastes the value as expected*

https://user-images.githubusercontent.com/16504501/140629932-c30ba94c-86d9-4287-8b82-2d61f53b910d.mov

## Auto-advance focused field
*after a field is filled, the form advances focus to the next input*

https://user-images.githubusercontent.com/16504501/140667183-6a67971f-b102-421a-a13e-4f9354243e96.mov

## Form error state

https://user-images.githubusercontent.com/16504501/140669623-f175621d-374e-4a82-a9d8-9c5dfa195272.mov

# Changes
- [x] When copying the master ticket, there shouldn't be a sig (`~`) at the front. The form where you input it doesn't accept a sig. 
- [x] Pasting the master ticket into the form should paste each of the four segments into the appropriate boxes
- [x] Entering the correct number of characters into a master ticket input should advance the cursor to the next input
- [x] Show an error message when entering an invalid ticket

closes #673, closes #700

### CODE REVIEW

**Review by willbach — APPROVED** [review]
lgtm

**Comment by jyng** [comment]
hold for review

**Comment by tomholford** [comment]
Thanks for the prompt review, @willbach. FYI snuck in one more commit that adds the form error state

File: src/views/Activate/MasterKeyConfirm.tsx [hunk 1]
```diff
@@ -161,7 +169,13 @@ const MasterKeyConfirm = () => {
                             fontFamily={'Source Code Pro'}
                             borderRadius={'5px'}
                             border={'solid 1px rgba(0,0,0,0.25)'}
+                            borderColor={
+                              showError ? '#FF4136' : 'rgba(0,0,0,0.25)'
```
Comment (willbach) [inline src/views/Activate/MasterKeyConfirm.tsx:173]: can you name these colors as constants?

**Review by willbach — COMMENTED** [review]
just the one comment: use constant names for the error colors

File: src/views/Activate/MasterKeyConfirm.tsx (hunk 1 above)
Comment (jyng) [reply src/views/Activate/MasterKeyConfirm.tsx:173]: shouldn't these colors be values from Indigo?

Comment (willbach) [reply src/views/Activate/MasterKeyConfirm.tsx:173]: yes, if Indigo exports colors as JS strings. I don't know enough about Indigo.

**Review by yosoyubik — APPROVED** [review]

**Pull request merged**